### ISSUE TITLE
Target size and target spacing not mentioned in WCAG 3

### ISSUE THREAD

**Issue by julierawe** [w3c/wcag3#808]
The [“Control Information” subsection](https://w3c.github.io/wcag3/guidelines/#control-information) of the Interactive Components section is currently missing provisions about target size and target spacing:

- **Target size** is very important for helping people with hand tremors, low vision, limited vision, and other challenges hit the target they want to hit and avoid accidentally hitting the wrong thing.
- **Target spacing** is very important for helping users avoid hitting another control that is too close to the one they were aiming for.

The current Interactive Components section does have a "[Focus indicator size sufficient](https://w3c.github.io/wcag3/guidelines/#focus-indicator-size-sufficient)" provision, but target size and focus size indicator serve different purposes. Target size ensures that interactive elements are easy to click/tap, while the focus indicator helps keyboard users track their location on the page. 

Also, please keep in mind this important [WCAG 2.2 note](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html) about [Target size (minimum)](https://www.w3.org/TR/WCAG22/#target-size-minimum): 
  “This success criterion defines a minimum size and, if this can't be met, a minimum spacing. It is still possible to have very small, and difficult to activate, targets and meet the requirements of this Success Criterion, provided that the targets don't have any adjacent targets that are too close. However, using larger target sizes will help many people use targets more easily. As a best practice it is recommended to at least meet the minimum size requirement of the Success Criterion, regardless of spacing. For important links/controls, consider aiming for the stricter [2.5.5 Target Size (Enhanced)](https://www.w3.org/WAI/WCAG22/Understanding/target-size-enhanced.html).”

**COGA user need missing or not adequately addressed -- from Making Content Usable for People with Cognitive and Learning Disabilities:** 

- I need the controls to be easy to correctly activate. The interface is designed so that I rarely activate controls by accident. ([MCU 3.1.2](https://www.w3.org/TR/coga-usable/#clear-operation-user-story), bullet 3)

With thanks,
The COGA Task Force

**Comment by GreggVan** [comment]
Question: if you don't know what size screen, something is going to be displayed on how do you have any idea or control over the size of the item. We can talk about things relative, but not in any actual physical size.  no?
